### PR TITLE
API-6368: repository alignment, scalastyle, scalafix, scalafmt.

### PR DESCRIPTION
**Pull request by mattclark-zerogravit**

### CODE REVIEW

**Comment by anjumabbas5** [comment]
There are some outstanding PR bot comments that can be addressed.

File: .scalafix.conf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  groupedImports = Merge
+  groups = [
+    "re:(javax?|scala)\\."
+    "---"
+    "*"
+    "---"
+    "play"
+    "uk.gov.hmrc."
+    "---"
+    "uk.gov.hmrc.hello."
```
Comment (anjumabbas5) [inline .scalafix.conf]: Needs changing to the repo

File: .scalafmt.conf [hunk 2]
```diff
@@ -1,36 +1,39 @@
-version = 2.6.4
+version="3.5.3"
+runner.dialect = "scala213"
```
Comment (anjumabbas5) [inline .scalafmt.conf]: does this need to be scala212?

**Review by anjumabbas5 — APPROVED** [review]

File: .scalafix.conf (hunk 1 above)
Comment (mattclark-zerogravit) [reply .scalafix.conf]: sorted

File: .scalafmt.conf (hunk 2 above)
Comment (mattclark-zerogravit) [reply .scalafmt.conf]: yeah have changed

**Pull request merged**